alerting/src/main/kotlin/org/opensearch/alerting/util/DocLevelMonitorQueries.kt

```diff
@@ -27,16 +33,17 @@ import org.opensearch.commons.alerting.model.DocLevelMonitorInput
 import org.opensearch.commons.alerting.model.DocLevelQuery
 import org.opensearch.commons.alerting.model.Monitor
 import org.opensearch.commons.alerting.model.ScheduledJob
+import org.opensearch.rest.RestStatus
 
 private val log = LogManager.getLogger(DocLevelMonitorQueries::class.java)
 
 class DocLevelMonitorQueries(private val client: Client, private val clusterService: ClusterService) {
     companion object {
 
-        val PROPERTIES = "properties"
-        val NESTED = "nested"
-        val TYPE = "type"
-
+        const val PROPERTIES = "properties"
+        const val NESTED = "nested"
+        const val TYPE = "type"
+        const val INDEX_PATTERN_SUFFIX = "-000001"
```

Comment:
Is the number getting incremented during rollovers? If so, where?

Reply:
It is done by _rollover core's API [here](https://github.com/petardz/alerting/blob/percolate-index-rollover/alerting/src/main/kotlin/org/opensearch/alerting/util/DocLevelMonitorQueries.kt#L376)

Reply:
Thanks